framework/src/st_things/things_stack/utils/things_ping.h

```diff
@@ -33,12 +37,39 @@ typedef enum ping_state_e {
 	PING_ALL_FLAG = 0xFFFFFFFF
 } ping_state_e;
 
+//-------------------------------------------------------------------------------
+/**
+ * @brief API for initialize OIC device keepAlive for ping resource.
+ * @return res success = true, otherwise false.
+ */
 bool things_ping_init(void);
 
+//-------------------------------------------------------------------------------
+/**
+ * @brief API for terminate OIC device keepALive for ping resource.When cloud module
```

Comment:
Please rephrase the sentences to present tense. Thank you.

Reply:
ok.